src/Analyser/NodeScopeResolver.php

```diff
@@ -1956,8 +1956,22 @@ function (MutatingScope $scope) use ($expr, $nodeCallback, $context): Expression
 						$nameType->getCallableParametersAcceptors($scope),
 					);
 				}
+
 				$nameResult = $this->processExprNode($expr->name, $scope, $nodeCallback, $context->enterDeep());
 				$throwPoints = $nameResult->getThrowPoints();
+				if (
+					$nameType->isObject()->yes()
+					&& $nameType->accepts(new ObjectType(Closure::class), true)->no()
```

Comment:
`accepts` isn't the right way to check for a type.

You need `(new ObjectType(Closure::class))->isSuperTypeOf($nameType)`.

Reply:
I updated the code with isSuperTypeOf

I always have difficulties to understand which method I should use.
Would it be possible to add comment with example above the method in the interface ?
Or at least explaining me the difference ?

Thanks.

Reply:
I agree that its not that easy to invoke the correct method from `Type`.

most of the time we need `isSuperTypeOf` though ;-)